test/Scheduling/shared-resources-problems.mlir

```diff
@@ -69,3 +69,26 @@ ssp.instance @multiple of "SharedOperatorsProblem" {
     operation<@_1> @last(%5) [t<11>]
   }
 }
+
+// CHECK-LABEL: if_else
+ssp.instance @if_else_exclusive of "SharedResourcesProblem" {
```

Comment:
Please add a couple more tests for multi-resource operations that contain potential resource conflicts. I'd like to see:
- Two ops competing for a set of common resources
- Limits > 1